### DISCUSSION TITLE
Student Pack benefits

### DISCUSSION THREAD

**Discussion by Nabeel-javed** [community/community#134384] [GitHub Education]
### Hi everyone,

I've applied for student pack and i got accepted now i'm waiting for the benefits and i want to use copilot but it is taking too much time, my application got accepted on 31st july and today its 31st August, how much time it would take?

**Comment by spaghettibernie** [comment]
Hey there!

If that were to be the case; that would have been an awful lot of time :)
I'm in the same boat as you; I was accepted on the 30th of July and today marks the **2nd of August**!

_(Yes, sorry! -- I'm piggybacking off you)_

Here's what they say on their FAQ's:

> We aim to provide you with your educational benefits within **3 days of this approval status**. However, during busy periods like the start of a new semester, it may take a bit longer.

> If you have not received your education benefits within **5 days of your approval status** please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟

Source: [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by ghost** [comment]
education of student
